package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@docusaurus/core": "^2.4.3",
     "@docusaurus/plugin-content-blog": "^2.4.3",
+    "@docusaurus/plugin-google-gtag": "^2.4.3",
```

Comment:
This line isn't needed if using the classic preset.

@Abhijay007 - see previous comment. Do we have an implementation preference?

Reply:
Yeah if we are using `@docusaurus/preset-classic ` preset then we don't need to install `@docusaurus/plugin-google-gtag`